lib/queries/addChallenge.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var fields = ['title', 'description', 'date', 'org_id', 'creator_id', 'active'];
+
+module.exports = function (objectToAdd) {
+  var values = fields.map(function (field) {
+    return '\'' + objectToAdd[field] + '\'';
```

Comment:
Is objectToAdd[field] always defined? It might be worth to check that the object contains all the properties and to return an error otherwise.